web/src/hooks/useContrastTheme.tsx

```diff
@@ -0,0 +1,5 @@
+import { useContext } from 'react'
+
+import { ThemeContext, ThemeContextType } from '../components/ThemeContext'
+
+export const useContrastTheme = (): ThemeContextType => useContext(ThemeContext)
```

Comment:
🔧 This is not contrast theme specific:
```suggestion
export const useThemeContext = (): ThemeContextType => useContext(ThemeContext)
```